### PR TITLE
Screen Options Admin Toggle SVG Icon

### PR DESCRIPTION
**Pull request by grayghostvisuals**
Removes the icon font reference for the screen options toggle. Since the state of the toggle is being controlled by JS I included a function in common.js to control an SVG icon swap so it can be used with other scenarios rather than rewriting the same task over and over again.

```javascript
function svgIconSwap(target, icon) {
	var el = $(target);
	el.attr('xlink:href', '/wp-includes/icons/dashicons.svg#' + icon);
}

var dashicon = '.dashicon use';
svgIconSwap(dashicon, 'dashicons-arrow-down');
svgIconSwap(dashicon, 'dashicons-arrow-up');
```

I also added the dashicons styles back in to be loaded by the script loader as I was not totally sure if plugins ref the styles being pulled in or not. If not we can remove them again.

Also note that alignment will still be off as this is a bigger issue to deal with in CSS once the icons are in place.

### CODE REVIEW

File: src/wp-admin/includes/class-wp-screen.php [hunk 1]
```diff
@@ -885,12 +885,22 @@ public function render_screen_meta() {
 		<div id="screen-meta-links">
 		<?php if ( $this->get_help_tabs() ) : ?>
 			<div id="contextual-help-link-wrap" class="hide-if-no-js screen-meta-toggle">
-			<button type="button" id="contextual-help-link" class="button show-settings" aria-controls="contextual-help-wrap" aria-expanded="false"><?php _e( 'Help' ); ?></button>
+				<button type="button" id="contextual-help-link" class="button show-settings" aria-controls="contextual-help-wrap" aria-expanded="false">
+					<?php _e( 'Help' ); ?>
+					<svg class="dashicon" aria-hidden="true">
```
Comment (ryelle) [inline src/wp-admin/includes/class-wp-screen.php]: Use `wp_icon` to output the SVG markup here -- `wp_icon( 'dashicons-arrow-down' );`

File: src/wp-admin/includes/class-wp-screen.php [hunk 2]
```diff
@@ -885,12 +885,22 @@ public function render_screen_meta() {
...
+					</svg>
+				</button>
 			</div>
 		<?php endif;
 		if ( $this->show_screen_options() ) : ?>
 			<div id="screen-options-link-wrap" class="hide-if-no-js screen-meta-toggle">
-			<button type="button" id="show-settings-link" class="button show-settings" aria-controls="screen-options-wrap" aria-expanded="false"><?php _e( 'Screen Options' ); ?></button>
+				<button type="button" id="show-settings-link" class="button show-settings" aria-controls="screen-options-wrap" aria-expanded="false">
+					<?php _e( 'Screen Options' ); ?>
+					<svg class="dashicon" aria-hidden="true">
```
Comment (ryelle) [inline src/wp-admin/includes/class-wp-screen.php]: Same here. The idea is we never write SVG markup, it should always be generated from `wp_icon`

File: src/wp-includes/script-loader.php [hunk 3]
```diff
@@ -825,6 +825,7 @@ function wp_default_styles( &$styles ) {
 
 	// Common dependencies
 	$styles->add( 'buttons',   "/wp-includes/css/buttons$suffix.css" );
+	$styles->add( 'dashicons', "/wp-includes/css/dashicons$suffix.css" );
```
Comment (ryelle) [inline src/wp-includes/script-loader.php]: Let's leave this to another PR.

File: src/wp-admin/includes/class-wp-screen.php (hunk 2 above)
Comment (grayghostvisuals) [inline src/wp-admin/includes/class-wp-screen.php]: Thanks for the pointer. I had a hunch it wasn't a good idea when I did this.

File: src/wp-includes/script-loader.php (hunk 3 above)
Comment (grayghostvisuals) [inline src/wp-includes/script-loader.php]: SGTM

File: src/wp-admin/includes/class-wp-screen.php (hunk 1 above)
Comment (grayghostvisuals) [inline src/wp-admin/includes/class-wp-screen.php]: 👍

**Comment by grayghostvisuals** [comment]
@ryelle There are a bit more additions to the PR; sorry about that. I will make better branches next time around as I work.

File: src/wp-admin/includes/class-wp-screen.php [hunk 4]
```diff
@@ -887,19 +887,15 @@ public function render_screen_meta() {
 			<div id="contextual-help-link-wrap" class="hide-if-no-js screen-meta-toggle">
 				<button type="button" id="contextual-help-link" class="button show-settings" aria-controls="contextual-help-wrap" aria-expanded="false">
 					<?php _e( 'Help' ); ?>
-					<svg class="dashicon" aria-hidden="true">
-		  			<use xlink:href="/wp-includes/icons/dashicons.svg#dashicons-arrow-down"></use>
-					</svg>
+					<?php _e( wp_icon( 'dashicons-arrow-down' ) ); ?>
```
Comment (ryelle) [inline src/wp-admin/includes/class-wp-screen.php]: You don't need to wrap this in `_e`, [_e](https://developer.wordpress.org/reference/functions/_e/) is a translation function. I swapped this for `echo` before I merged the PR, but for future PRs you can just `echo wp_icon()`

File: src/wp-admin/js/common.js [hunk 5]
```diff
@@ -1,5 +1,12 @@
 /* global setUserSetting, ajaxurl, commonL10n, alert, confirm, pagenow */
 var showNotice, adminMenu, columns, validateForm, screenMeta;
+
+// so we have global access with other scripts
+function svgIconSwap( target, icon ) {
```
Comment (ryelle) [inline src/wp-admin/js/common.js]: I think we'll want to revisit this function later — probably try to find a place for it in the `wp` object.
